### DISCUSSION TITLE
#3329 issue manual fix or budibase upgrade in docker

### DISCUSSION THREAD

**Discussion by itworksmart** [Budibase/budibase#3850] [Help]
I use budibase version 0.9.173 through docker.

I found the same issue in #3329 today.
https://github.com/Budibase/budibase/discussions/3329

First I create a app in budibase, then I create a new budibase user, but I dont found user in app user list.
And I cannot find add user in user list.

Would like to know if I can use command to update in docker or not?

Or how can I fix user issue by myself?

**Comment by mjashanks** [comment]
Hi @itworksmart - you should take an update. I believe that this is fixed. Take an update!

**Reply by itworksmart** [reply]
Can you kindly guide me how to upgrade?
I can run script in docker image, or I need to establish a new docker app

As far as I know, I can export a app (schema), but db data can only export not seen import option.
So I don't know how to completely backup or shift total app (include data inside) to other budibase installation.

**Reply by mjashanks** [reply]
It depends on how you installed it. Start from the directory where your config files are stored, then

- If you installed via the budibase cli, e.g. `budi hosting --init` then you should run `budi hosting --update` 
- if you installed via `docker-compose`, see this comment: https://github.com/Budibase/budibase/discussions/1198#discussioncomment-408555
